.github/workflows/ubuntu.yml

```
@@ -22,6 +22,21 @@ on:
     branches-ignore:
       - 'gh-pages'
 
+    tags: []
```

actionlint (1.7.7)

"tags" section should not be empty

(syntax-check)